inc/changelog.php

```diff
@@ -743,6 +743,15 @@ public function getRevisionsAround($rev1, $rev2, $max = 50) {
 
         return array(array_reverse($revs1), array_reverse($revs2));
     }
+    
+    /**
+     * Checks if the ID has old revisons
+     * @return boolean
+     */
+    public function hasRevisions() {
+        $file = $this->getChangelogFilename();
+        return file_exists($file);
```

Comment:
Could you add unit tests to cover this method? see for examples of other changelog tests in https://github.com/splitbrain/dokuwiki/tree/master/_test/tests/inc (the changelog_*.test.php files)

Reply:
just added some tests, are they enough?

Reply:
Yes, because deleting a page will only add a line to this file. Thanks.